src/extension.ts

```diff
@@ -110,4 +112,81 @@ export const activate = () => {
         if (!getExtensionSetting('runOnSave') || reason === vscode.TextDocumentSaveReason.AfterDelay) return
         waitUntil(performFixes())
     })
+
+    vscode.workspace.onDidChangeTextDocument(
+        ({ contentChanges, document }) => {
+            if (!getExtensionSetting("insertComma")) {
+                return;
+            }
+
+            if (
+                document.languageId !== "json" &&
+                document.languageId !== "jsonc"
+            ) {
+                return;
+            }
+
+            if (contentChanges.length === 0) {
+                return;
+            }
+
+            const editor = vscode.window.activeTextEditor;
+
+            if (
+                document.uri !== editor?.document.uri ||
+                ["output"].includes(editor.document.uri.scheme)
+            ) {
+                return;
+            }
+            if (
+                vscode.workspace.fs.isWritableFileSystem(
+                    document.uri.scheme
+                ) === false
+            ) {
+                return;
+            }
+
+            const content = contentChanges[0];
```

Comment:
Also let's support multicuror editing. Wrap code below into `editor.edit()` and then into a loop. Like so:

```ts
void editor.edit((edit) => {
    for (const contentChange of contentChanges) {
        // basically the same code as below but with continue instead of return:
        // do edit.insert if needed on specific contentChange
    }
})
```